debian/gbp.conf

```diff
@@ -0,0 +1,3 @@
+[DEFAULT]
+debian-branch=ev3dev-stretch
+debian-tag=ev3dev-stretch/%(version)s
```

Comment:
I changed the branch to `stretch` to match the repo. It might make more sense to switch the branch names to `ev3dev-{stretch, jessie}` to match other repos, but I'm don't particularly care.

Reply:
Making them all the same (`ev3dev-stretch`) would be my inclination, but you have the parallel 1.x and 2.x branches here, so calling this branch `stretch-2.x` might make more sense, perhaps?

Reply:
Would you be opposed to `ev3dev-stretch-1.x` and `ev3dev-stretch`? Or perhaps `ev3dev-stretch-legacy` for the 1.x series? My hope is that I can avoid typing `-2.x` if it isn't necessary.

Reply:
First option sounds fine. (If you make `ev3dev-stretch` the default branch on GitHub, you should never have to type it in anyways 😉 .)

Reply:
> First option sounds fine. 

I just pushed the new branches and deleted the old `stretch[-1.x]` branches. I don't have admin access on this repo though; could you switch the default branch to `ev3dev-jessie`?

> If you make ev3dev-stretch the default branch on GitHub, you should never have to type it in anyways 😉

We'll do that once we release the first stretch version, I think. If everyone's running code from the jessie branch I'd rather make that the default -- once there's Stretch code out in the wild that changes.